### PR TITLE
added image for matcha-green-tea-smoothie

### PR DESCRIPTION
**Pull request by Kirti-Motwani**

### CODE REVIEW

File: src/recipes/matcha-green-tea-smoothie.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        {
+            "quantity": "1",
+            "measure": "cup",
+            "ingredient": "light coconut milk"
+        }
+    ],
+    "directions": [
+        "Add the coconut milk to your blender jug, followed by the matcha green tea powder and spinach. Blend.",
+        "Add the mango and frozen bananas and blend until creamy and smooth.",
```
Comment (alfg) [inline src/recipes/matcha-green-tea-smoothie.json]:
```suggestion
        "Add the mango and frozen bananas and blend until creamy and smooth."
```

**Comment by alfg** [comment]
Thanks!

**Comment by Kirti-Motwani** [comment]
Thanks alot! Can you help me? I got one question!

**Comment by alfg** [comment]
@Kirti-Motwani Sure, what's the question?

**Pull request merged**